cou/apps/auxiliary.py

```diff
@@ -42,17 +44,9 @@ def possible_current_channels(self) -> list[str]:
         :rtype: list[str]
         """
         tracks = OPENSTACK_TO_TRACK_MAPPING.get(
-            (self.charm, self.series, self.current_os_release.codename)
-        )
-        if tracks:
-            return [f"{track}/stable" for track in tracks]
-
-        raise ApplicationError(
-            (
-                f"Cannot find a suitable '{self.charm}' charm channel for "
-                f"{self.current_os_release.codename}"
-            )
+            (self.charm, self.series, self.current_os_release.codename), []
```

Comment:
I think we should remove `:raises ApplicationError: When cannot find tracks.` since this will no longer raise any exception.